deploy/crds/keycloak.org_keycloaks_crd.yaml

```diff
@@ -44,6 +44,15 @@ spec:
                   description: If set to true, the Operator will create an Ingress
                     or a Route pointing to Keycloak.
                   type: boolean
+                tlsTermination:
```

Comment:
From what I know OpenShift uses "reencrypt", "passthrough" and "edge". We should not support "edge" at all.

What does the value "router" mean? I actually don't understand if you mean "edge" or "reencrypt" here. I would just use the standard OpenShift names for it.

What's the default for this? I think we said "reencrypt" and I see in the code below that's what's used if the value is not set?! shouldn't the default behaviour be mentioned here?

Why do we not set the termination on Kubernetes? Do we not support setting up the Ingress at all?

Reply:
> From what I know OpenShift uses "reencrypt", "passthrough" and "edge". We should not support "edge" at all.

And we don't. We support only re-encrypt and passthrough.

> What does the value "router" mean? I actually don't understand if you mean "edge" or "reencrypt" here. I would just use the standard OpenShift names for it.

I didn't want to use re-encrypt, since it's an implementation specific term. But after sleeping on it, I changed my mind.

> What's the default for this? I think we said "reencrypt" and I see in the code below that's what's used if the value is not set?! shouldn't the default behaviour be mentioned here?

Re-encrypt. Ok, I'll mention it.

> Why do we not set the termination on Kubernetes? Do we not support setting up the Ingress at all?

Yes, we do setup an ingress. However, frontend TLS termination requires creating a Secret with key/cert pair and referencing it in the Ingress. We don't have any way of generating those. They need to be supplied by a user. See the [documentation](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls).